integrations/MoEngage/browser.js

```diff
@@ -0,0 +1,184 @@
+import each from "@ndhoule/each";
+import logger from "../../utils/logUtil";
+
+// custom traits mapping context.traits --> moengage properties
+const traitsMap = {
+  firstName: "first_name",
+  lastName: "last_name",
+  firstname: "first_name",
+  lastname: "last_name",
+  email: "email",
+  phone: "mobile",
+  name: "user_name",
+  username: "user_name",
```

Comment:
As we're supporting camelCase, should we support `userName` as well?

Reply:
done